setup.py

```diff
@@ -0,0 +1,34 @@
+import distutils.core
+from distutils.command.install import install
+import subprocess
+
+
+class PyenvInstall(install):
+    def run(self):
+        print(subprocess.check_output(['bash', 'bin/pyenv-installer']))
+
+with open('README.rst') as file:
+    long_description = file.read()
+
+distutils.core.setup(
+    version='20150113',
```

Comment:
ahh ... I just added todays date as version number for the initial release. As the installer as such has no internal version number (e.g. in the file itself) this place in setup.py would be the single place where were the version number has to be maintained.

I will have a look tonight how it works on PyPi to add you as package owner, so that you can do releases on PyPi, otherwise I am ok to handle that part as well.